### ISSUE TITLE
Cannot read property 'setup' of undefined

### ISSUE THREAD

**Issue by kfamilonidis** [segmentio/analytics-react-native#4]
Following the documentation in read me I get this error:
```
Cannot read property 'setup' of undefined
```

**Comment by f2prateek** [comment]
hey @kfamilonidis can you show the code that you're using and clarify where you're seeing this error (e.g. is this in your editor, or a runtime error)?

**Comment by kfamilonidis** [comment]
@f2prateek  the code I'm calling is: 
```
import analytics from '@segement/analytics-react-native'

analytics.configure().setup('my_key')
```

The error is at runtime in browser's console.

**Comment by fathyb** [comment]
Hey @kfamilonidis, the error you're getting probably stems from missing native modules. We've just [released `0.0.1-beta.1`](https://github.com/segmentio/analytics-react-native/blob/master/CHANGELOG.md#001-beta1-2018-10-23) with an improved error message. Could you try it and show us if the erroor message changes?

We've also added some [trobleshooting instructions in the `README.md`](https://github.com/segmentio/analytics-react-native#failed-to-load--native-module) for the kind of error, let us know if it helped!

**Comment by fathyb** [comment]
Closing this issue due to inactivity. Feel free to re-open if you have any questions!
